docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,39 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance), making them more precise.
+You can supply the \code{-AignoreRangeOverflow} command-line option
+when you are confident that no overflow will occur in the program (for
```

Comment:
@kelloggm What is the tradeoff between writing upper bound annotations and using `-AignoreRangeOverflow`?  That is, would it always be possible to write upper bound annotations instead of using `-AignoreRangeOverflow`?  Is the reason for the command-line argument that it too much of a hassle to write the upper bound annotations, or is there a different motivation?